rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -39,10 +39,19 @@ namespace rsx
 	{
 		//Don't throw, gather information and ignore broken/garbage commands
 		//TODO: Investigate why these commands are executed at all. (Heap corruption? Alignment padding?)
+		rsx->invalid_command_interrupt_raised |= !(_reg >= 0xac80 >> 2 && _reg < 0xb000 >> 2); // HACK: Allow these "unknown" methods
+ 
 		const u32 cmd = rsx->get_fifo_cmd();
-		rsx_log.error("Invalid RSX method 0x%x (arg=0x%x, start=0x%x, count=0x%x, non-inc=%s)", _reg << 2, arg,
-		cmd & 0xfffc, (cmd >> 18) & 0x7ff, !!(cmd & RSX_METHOD_NON_INCREMENT_CMD));
-		rsx->invalid_command_interrupt_raised = true;
+		if (rsx->invalid_command_interrupt_raised || _reg == 0xac80 >> 2)
```

Comment:
Why are you only checking for ac80?

Reply:
Avoiding log spam, I will add a better solution for here that logs once the first invalid method from all this range even if it wasn't ac80.